modules/fedimint-mint-client/src/lib.rs

```diff
@@ -74,14 +74,27 @@ const MINT_BACKUP_RESTORE_OPERATION_ID: OperationId = OperationId([0x01; 32]);
 
 pub const LOG_TARGET: &str = "client::module::mint";
 
+#[derive(Clone, Debug, Decodable, Encodable, Deserialize, Serialize)]
+pub struct OOBNotes {
+    federation_id: FederationId,
+    pub notes: TieredMulti<SpendableNote>,
+}
```

Comment:
Why is one `pub` but not the other?

Reply:
I initially exposed `notes` for `totalAmount()` but then decided to make a pass-through function on `OOBNotes` itself. I suppose they could both be non-`pub` and there could be getters for both fields? Or they could both be made `pub`? Probably the latter is good enough?

Reply:
I'd make both `pub`. This struct is just a no-logic aggregator of two datums.